### PR TITLE
feat: Text Rating Navigation components

### PR DESCRIPTION
**Pull request by DevNirah7**
Text Utilities, Rating and Navigation component PR

Live preview link : https://deploy-preview-13--mint-ui-library.netlify.app/src/components/navigation/navigation

### CODE REVIEW

File: src/components/avatars/avatars.html [hunk 1]
```diff
@@ -49,6 +49,15 @@
                         <li>
                             <a href="/src/components/inputs/inputs.html">Input</a> 
                         </li>
+                        <li>
+                            <a href="/src/components/inputs/inputs.html">Rating</a> 
+                        </li>
```
Comment (vikrantkode) [inline src/components/avatars/avatars.html]: wrong link in rating

File: src/components/navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                        <li>
+                            <a href="/src/components/badges/badges.html">Badge</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/buttons/buttons.html">Button</a> 
+                        </li>
+                        <li>
+                            <a href="">Card</a> 
+                        </li>
+                        <li>
```
Comment (vikrantkode) [inline src/components/navigation/navigation.html:47]: add card link here

**Review by vikrantkode — CHANGES_REQUESTED** [review]
- Rating link is not there in any html . Please add them
- Add Card link also
-Rest looks good to me.

File: src/components/avatars/avatars.html (hunk 1 above)
Comment (DevNirah7) [reply src/components/avatars/avatars.html]: Updated the links

File: src/components/navigation/navigation.html (hunk 2 above)
Comment (DevNirah7) [reply src/components/navigation/navigation.html:47]: Card component is done in a separate branch, will add once done

File: src/components/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                      <li><i class="fas fa-lg fa-home"></i></li>
+                      <li><i class="fab fa-lg fa-gratipay"></i></li>
+                      <li><i class="fas fa-lg fa-shopping-cart"></i></li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="code-wrapper">
+                <iframe
+                src="https://carbon.now.sh/embed/8Pz8PQKs8F1dHRiBDYiE"
+                style="width: 708px; height: 334px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (shubhmehra) [inline src/components/navigation/navigation.html:86]: you can use width : 100% for responsiveness

File: src/components/ratings/ratings.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+{
+    padding: var(--u-4 0);
+
+}
+
+.rating-div > i
+{
+    margin: 0 var(--u-2);
+    color: var(--gold);
+}
```
Comment (shubhmehra) [inline src/components/ratings/ratings.css:11]: please use prettier to code formating

**Review by shubhmehra — APPROVED** [review]
Code needs to be formatted!

File: src/components/navigation/navigation.html (hunk 3 above)
Comment (DevNirah7) [reply src/components/navigation/navigation.html:86]: Noted

**Pull request merged**